File: Algorithms/Strings/Game_of_Thrones_1/main.cpp
// Game of Thrones - I - Solution

#include <iostream>
#include <algorithm>

void gameOfThrones(std::string s) {
    std::sort(s.begin(), s.end());
    int count{1}, oddCount{};
    for (int i{}; i < s.size()-1; i++) {
        if (s[i] == s[i+1])
            count++;
        else {
            if (count % 2 != 0)
                oddCount++;
            count = 1;
        }
    }
    if (oddCount > 1)
        std::cout << "NO";
    else
        std::cout << "YES";      
}

int main() {
    std::string s;
    std::cin >> s;
    gameOfThrones(s);

    return 0;
}

File: Algorithms/Strings/Game_of_Thrones_1/main.py
### Game of Thrones - I - Solution

def gameOfThrones(s):
    sorted_s = tuple(sorted(s))
    count, oddCount = 1, 0
    for i in range(len(sorted_s)-1):
        if sorted_s[i] == sorted_s[i+1]:
            count += 1
        else:
            if count % 2 != 0:
                oddCount += 1
            count = 1
    if oddCount > 1:
        print("NO")
    else:
        print("YES")

s = input()
gameOfThrones(s)